src/app/Main/Body/QueryPage/NumberOfIsolates/countNrOfIsolates.component.ts

```diff
@@ -0,0 +1,5 @@
+import { DBentry } from "../../../../Shared/Isolat.model";
+
+export function countNrOfIsolates(data: DBentry[]): string {
+    return (Object.keys(data).length as unknown) as string;

```

Comment:
- why not "return String(data.length)"
- file is named like it is a component but it exports only a helper function
- i would remove this file because only purpose is to get the length of an array

Reply:
Thanks, you are absolutely right.